SECP-service/src/test/java/com/visucius/secp/models/GroupTest.java

```diff
@@ -0,0 +1,59 @@
+package com.visucius.secp.models;
+
+import org.junit.Test;
+
+import javax.persistence.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class GroupTest {
+
+
+    @Test
+    public void testGroupForEntityAndTableAttribute() {
+        AssertAnnotations.assertType(Group.class, Entity.class, Table.class);
+    }
+
+    @Test
+    public void testId(){
+        //Testing annotations on the id field
+        AssertAnnotations.assertField(Group.class, "id", Id.class, GeneratedValue.class, Column.class);
+
+        //Testing the @column annotation
+        Column c = ReflectTool.getFieldAnnotation(Group.class, "id", Column.class);
+
+        assertEquals("column id: name is not equal","id", c.name());
+        assertEquals("column id: unique is false",true, c.unique());
+        assertEquals("column id: nullable is true",false, c.nullable());
+    }
+
+    @Test
+    public void testName(){
+        AssertAnnotations.assertField(Group.class, "name", Column.class);
+
+        Column c = ReflectTool.getFieldAnnotation(Group.class, "name", Column.class);
+
+        assertEquals("column name: name is not equal", "name", c.name());
+        assertEquals("column name: unique is false", true, c.unique());
+        assertEquals("column name: nullable is true", false, c.nullable());
+    }
+
```

Comment:
The test case for the group_roles table is missing. Please kindly include it. :)